### ISSUE TITLE
Support aggregating stats of the same job from selected Scrapyd servers

### ISSUE THREAD

**Issue by my8100** [my8100/scrapydweb#72]

**Comment by my8100** [comment]
![image](https://user-images.githubusercontent.com/35719635/61768654-a1e47f00-ae1a-11e9-9654-ba8c54ccd535.PNG)

_Originally posted by @heave-Rother in https://github.com/my8100/scrapydweb/issues/4#issuecomment-514491171_

**Comment by my8100** [comment]
It could be named as the Reports page.

**Comment by my8100** [comment]
@heave-Rother @zhangshengchun
Would you like to try out the newly added pages for aggregating stats on your servers?
It would be appreciated if you could give me any feedback or suggestion.

To get the latest version from git:
```
pip install -U git+https://github.com/my8100/scrapydweb.git
```

* reports of all jobs from one server
![node_reports](https://user-images.githubusercontent.com/37427675/62120116-91894400-b2f3-11e9-97ae-0a60812a07b8.png)

* reports of one job from all servers
![cluster_reports](https://user-images.githubusercontent.com/37427675/62120219-c1d0e280-b2f3-11e9-9b02-4926765320bc.png)

**Comment by my8100** [comment]
Fixed in PR #75.
